src/components/data/DataTable/__stories__/DataTable.stories.tsx

```diff
@@ -121,6 +121,51 @@ const WithoutSkeleton: FC = () => {
         />
     );
 };
+const DefaultWithOwnStyle: FC = () => {
+    const [appSettings, setAppSetting] = useAppSettings();
+    const perPage = parseInt(appSettings['perPage'], 10) || 20;
+
+    const { sortedData, setSortBy, sortBy, direction } = useSorting(data, null, null);
+    const { pagination, pagedData, setCurrentPage } = usePagination(
+        sortedData as DataItem[],
+        perPage
+    );
+
+    const onPaginationChange = useCallback((newPage: Page, perPage: number) => {
+        setCurrentPage(newPage.index, perPage);
+        setAppSetting('perPage', perPage);
+    }, []);
```

Comment:
Should'nt the useCallback have dependencies or is it ment to not have any depencies that updates the funtion?

Reply:
Dep on func?
I just copy the other func in this story file.